nikolai4D/ContechOS
# C2-176 break filterbox overflow

## Description & motivation



filterbox titles now wrap inside the box

## How to test


Go to filter view, enter a long title def node, make sure it wraps in the filterbox.


## New tickets to be added





---
